pkg/virt-controller/services/template_test.go

```diff
@@ -2784,6 +2784,7 @@ var _ = Describe("Template", func() {
 
 				caps := pod.Spec.Containers[0].SecurityContext.Capabilities
 
+				Expect(caps.Add).To(Not(ContainElement(k8sv1.Capability("NET_ADMIN"))), "Compute container should not have NET_ADMIN capability")
```

Comment:
I think it is suppose to be `Expect(...).ToNot(ContainElement(...))`

But I really do not see how a tun device has any relation to having the capability.
It is probably some leftover from the time we had a priv root pod.

Today virt-launcher should never use this capability and we do not set it.
So I see no reason to ask about it, especially in this tun context.